contracts/MintBurnERC20.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract MintBurnERC20 is ERC20, AccessControl {
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
```

Comment:
The admin role is defined like this: `bytes32 public constant DEFAULT_ADMIN_ROLE = 0x00;`
Maybe have a smaller role definition for minter also? (will save some deployment gas in case somebody (us) will us this contract/

Reply:
https://docs.openzeppelin.com/contracts/5.x/access-control This is how it is done also in the docs. I would keep it as it is even if we would save some gas